### PR TITLE
fix 1.4.0 test

### PR DESCRIPTION
**Pull request by Stephen1993**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by zhengfeiwang — DISMISSED** [review]

File: src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def test_pf_test_with_additional_includes(self, caplog):
         assert all([key in FLOW_RESULT_KEYS for key in result])
 
         # Test additional includes don't exist
-        with pytest.raises(ValueError) as e:
+        with pytest.raises(UserErrorException) as e:
```
Comment (zhengfeiwang) [inline src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py:130]: this test will be skipped in any condition, so maybe no need to change this.

**Review by crazygao — APPROVED** [review]

File: src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py [hunk 2]
```diff
@@ -427,7 +427,7 @@ def test_run_with_additional_includes(self, pf, runtime: str, randstr: Callable[
         assert run.status == RunStatus.COMPLETED
 
         # Test additional includes don't exist
-        with pytest.raises(ValueError) as e:
+        with pytest.raises(UserErrorException) as e:
```
Comment (zhengfeiwang) [inline src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py:430]: this is not failed in current matrix test, may take a look on this

**Review by brynn-code — DISMISSED** [review]

File: src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py (hunk 2 above)
Comment (zhengfeiwang) [reply src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py:430]: this test will fail during local live test, need some change

File: src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py (hunk 1 above)
Comment (zhengfeiwang) [reply src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py:130]: this comment is optional, maybe take a look on below first

**Review by zhengfeiwang — APPROVED** [review]

File: src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py (hunk 1 above)
Comment (Stephen1993) [reply src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py:130]: Good catch, this error was thrown from SDK V2, it is indeed ValueError.

Comment (Stephen1993) [reply src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py:130]: I will modify it first to ensure that the code is correct.

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**